app.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express')
+const app = express()
+const {getAllTopicsData}= require('./controllers/topics.controller.js')
+const {getAllEndPointsInfo}= require('./controllers/endpoints.controller.js')
+const {getArticleByID, getAllArticles, getCommentsForArticle, postCommentForArticle, patchArticle}= require('./controllers/articles.controllers.js')
+const {deleteComment}= require('./controllers/comments.controller.js')
+app.use(express.json())
+
+app.get('/api/topics', getAllTopicsData)
+app.get('/api', getAllEndPointsInfo)
+app.get('/api/articles/:article_id', getArticleByID)
+app.get('/api/articles', getAllArticles)
+app.get('/api/articles/:article_id/comments', getCommentsForArticle)
+app.post('/api/articles/:article_id/comments', postCommentForArticle)
+app.patch('/api/articles/:article_id', patchArticle)
+app.delete('/api/comments/:comment_id', deleteComment)
+
+
+app.use((err, req, res, next) => {
+
+    if (err.msg && err.status) {
+        res.status(err.status).send(err)
+    } else {
+        next(err)
+    }
+})
+
+module.exports = app
```

Comment:
You might also benefit from handling PSQL errors. Apart from the link to the notes I already linked, this might be a good read too generally to get a better understanding on how we want to use our .catch blocks https://notes.northcoders.com/courses/js-back-end/advanced-error-handling